packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -143,9 +144,19 @@ export abstract class LibraryCommandletExecutor<T>
    * @param response Data from the parameter gathering step.
    * @returns Whether or not the execution was a success
    */
-  public abstract run(response: ContinueResponse<T>): Promise<boolean>;
+  public abstract run(
+    response: ContinueResponse<T>,
+    progress?: vscode.Progress<{
+      message?: string | undefined;
+      increment?: number | undefined;
+    }>,
+    token?: vscode.CancellationToken
+  ): Promise<boolean>;
 
   public async execute(response: ContinueResponse<T>): Promise<void> {
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
```

Comment:
we can remove line 157 & 158 as tokens are provided in `vscode.window.withProgress`